bodhi/tests/server/functional/test_generic.py

```diff
@@ -101,6 +103,81 @@ def test_home(self):
         self.assertIn('Log out', res)
         self.assertIn('Fedora Update System', res)
 
+    def test_home_counts(self):
+        user2 = User(name=u'dudemcpants')
+        self.db.flush()
+        self.db.add(user2)
+        release = self.db.query(Release).filter_by(name=u'F17').one()
+        addedupdates = [[UpdateStatus.pending,
+                         [[UpdateType.security, 5],
+                          [UpdateType.bugfix, 4],
+                          [UpdateType.enhancement, 3],
+                          [UpdateType.newpackage, 2]]],
+                        [UpdateStatus.testing,
+                         [[UpdateType.security, 15],
+                          [UpdateType.bugfix, 14],
+                          [UpdateType.enhancement, 13],
+                          [UpdateType.newpackage, 12]]],
+                        [UpdateStatus.stable,
+                         [[UpdateType.security, 25],
+                          [UpdateType.bugfix, 24],
+                          [UpdateType.enhancement, 23],
+                          [UpdateType.newpackage, 22]]]]
+        count = 0
+        for i in addedupdates:
+            for j in i[1]:
+                for k in range(0, j[1]):
+                    update = Update(
+                        title=u'bodhi-2.0-1%s.fc17' % (str(count)),
+                        user=user2,
+                        status=i[0],
+                        type=j[0],
+                        notes=u'Useful details!',
+                        release=release,
+                        date_submitted=datetime(1984, 11, 02),
+                        requirements=u'rpmlint',
+                        stable_karma=3,
+                        unstable_karma=-3,
+                    )
+                    self.db.add(update)
+                    self.db.flush()
+                    count = count + 1
+
+        res = self.app.get('/', status=200)
+        self.assertIn(
+            '<a href=\"http://localhost/updates/?releases=F17&amp;status=pending\">\n'
+            '          15\n'
+            '          </a>',
+            res
+        )
+        self.assertIn('<h4>updates pending</h4>', res)
+        self.assertIn('<span class="fa fa-shield"></span> 5', res)
+        self.assertIn('<span class="fa fa-bug"></span> 5', res)
```

Comment:
Added a comment here to say that this is one extra than what we expect, because the base test code already adds an update to the system